CHANGES.md

```diff
@@ -1191,7 +1191,79 @@ OpenSSL 3.0
 OpenSSL 1.1.1
 -------------
 
-### Changes between 1.1.1e and 1.1.1f [xx XXX xxxx]
+### Changes between 1.1.1h and 1.1.1i [xx XXX xxxx]
+
+ *
+
+### Changes between 1.1.1g and 1.1.1h [22 Sep 2020]
+
+ * Certificates with explicit curve parameters are now disallowed in
+   verification chains if the X509_V_FLAG_X509_STRICT flag is used.
+   [Tomas Mraz]
```

Comment:
Ups, I overlooked to replace some of the brackets around the names. Will do it later, no time left.

Reply:
Done